flask_app/tasks.py

```diff
@@ -91,10 +97,22 @@ def create_key(s: str) -> str:
     f.seek(0)
     return RSAKey.from_private_key(file_obj=f, password=None)
 
-with open(get_combadge_path('v1'), "rb") as combadge_v1:
-    _COMBADGE_V1 = combadge_v1.read()
-with open(get_combadge_path('v2'), "rb") as combadge_v2:
-    _COMBADGE_V2 = combadge_v2.read()
+
+def _download_combadge_assets() -> None:
+    if not os.path.exists(COMBADGE_ASSETS_DIR):
+        os.makedirs(COMBADGE_ASSETS_DIR)
+    for combadge_type in ['linux', 'darwin', 'windows']:
+        combadge_url = f'https://github.com/getslash/scotty/releases/latest/download/combadge_{combadge_type}'
+        response = requests.get(combadge_url, allow_redirects=True)
+        combadge_path = get_combadge_path(combadge_type)
+        with open(combadge_path, "wb") as combadge_file:
+            combadge_file.write(response.content)
+        combadge_st = os.stat(combadge_path)
+        os.chmod(combadge_path, combadge_st.st_mode | stat.S_IEXEC)
+        logger.debug(f"Updated combadge {combadge_type} asset. Modification time: {response.headers['Last-Modified']}")
+
+
+_download_combadge_assets()
```

Comment:
The combadge binary should be included in the distribution. If they are downloaded from GitHub the version might mismatch, among other issues this could cause (internet connectivity problems, GitHub downtime etc).

Reply:
The combadge build was moved to docker file